src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -92,8 +88,14 @@ winrt::com_ptr<Profile> Profile::CopySettings(winrt::com_ptr<Profile> source)
     profile->_ConnectionType = source->_ConnectionType;
     profile->_Origin = source->_Origin;
 
-    // Copy over the appearance
+    // Copy over the font info
     const auto weakRefToProfile = weak_ref<Model::Profile>(*profile);
+    winrt::com_ptr<FontConfig> sourceFontInfoImpl;
+    sourceFontInfoImpl.copy_from(winrt::get_self<FontConfig>(source->_FontInfo));

```

Comment:
If you do go with my pointer/reference suggestion you can get rid of this `winrt::com_ptr` dance.

Reply:
Same as above! Maintaining consistency with how we do Profile copy and Appearance copy